recipes/limix/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "limix" %}
+{% set version = "0.8.3" %}
+{% set sha256 = "5c025ed38b84bfd52373d9a2f4758903903be50f88da5d553dfd27c40791000d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: python setup.py install --compatible --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython
+    - numpy
```

Comment:
If you are going to build against numpy, you need to use `numpy x.x` http://conda-forge.github.io/docs/meta.html#building-against-numpy